cmd/format-meta.go

```diff
@@ -49,4 +49,6 @@ type formatMetaV1 struct {
 	Version string `json:"version"`
 	// Format indicates the backend format type, supports two values 'xl' and 'fs'.
 	Format string `json:"format"`
+	// ID is the identifier for the minio deployment
+	ID string `json:"id"`
```

Comment:
Isn't this redundant  with deployment id in formatXLV3 ?

Reply:
The fix for this is part of https://github.com/minio/minio/pull/6134 . Once that PR is merged, you should not see this issue